### PR TITLE
Adding Hover effect on Genre Cards

### LINKED ISSUE #157: feat: Adding hover effect to cards in "Read Genre Wise"
### Is your feature request related to a problem? Please describe.
The hover effect is given to "Read Genre Wise" is given to complete div which doesn't seem to be interactive for the users.

### Describe the solution you'd like
It would be nice if we give a hover effect to each card instead. I don't know it's by mistake or intentional but if you feel the same that it should be corrected, Assign it to be happy to work on it.

### Additional context

image

Here when we hover on it complete div is elevating.

### PR DESCRIPTION
**Pull request by iampranavdhar**
## Description
- Initially, there was a bug in  "Read Genre Wise"  which made complete div transform. Now I had changed that and had made a hover effect that will affect only the genre card. And I had considered all the widths and made the hover effects accordingly.
- Fixes #157 

## Affected Dependencies
NA

## How has this been tested?
This has been tested by seeing the output in the browser.

## GIF Before

prev

## GIF After

after

Hope everything is clear.
Thanks.

### CODE REVIEW

**Review by Praful932 — APPROVED** [review]
Seems fine to me 🚀

File: static/mainapp/css/index.css [hunk 1]
```diff
@@ -301,6 +304,15 @@ ol.carousel-indicators li.active {
 }
 /* media query*/
 
+@media screen and (max-width: 575px){
+  .col-lg-3:hover{
+    transform: scale(1.03);
+    transition: all 0.5s ease;
```
Comment (maneprajakta) [inline static/mainapp/css/index.css:310]: Only transform needs to be handled by media query no need to repeat the remaining attributes

**Review by maneprajakta — CHANGES_REQUESTED** [review]

**Comment by iampranavdhar** [comment]
Had Done the requested changes. Can you look into it @maneprajakta

**Review by maneprajakta — APPROVED** [review]
Great :rocket:

**Pull request merged**